isic/login/views.py

```diff
@@ -14,3 +16,12 @@ def get_girder_token(request):
         raise Exception('Profile has no girder_id', request.user.profile)
     token = create_girder_token(request.user.profile.girder_id)
     return JsonResponse({'token': token})
+
+
+@swagger_auto_schema(methods=['PUT'], operation_summary='Accept the terms of use.')
+@api_view(['PUT'])
+@permission_classes([IsAuthenticated])
+def accept_terms_of_use(request):
+    request.user.profile.accepted_terms = timezone.now()
```

Comment:
If it's already accepted, this shouldn't update the DB. This can probably be done with a fancy DB query.